airbyte-integrations/connectors/destination-typesense/destination_typesense/destination.py

```diff
@@ -15,8 +15,8 @@
 def get_client(config: Mapping[str, Any]) -> Client:
     api_key = config.get("api_key")
     host = config.get("host")
-    port = config.get("port") or "8108"
-    protocol = config.get("protocol") or "https"
```

Comment:
The default port was previously 8108 with https. Is 443 a better default?

Reply:
I'm not sure we shall force HTTPS on 443 as users with HTTPS instance not on 443 won't be able to use the connector.

Reply:
good call, I'll keep the option to set custom port, will change that now

Reply:
> The default port was previously 8108 with https. Is 443 a better default?
8108 is a default port for local Typesense instance, thought this connector would be used primarily on cloud, but for the sake of saving ability to run syncs locally I'll keep it as well